HISTORY.md

```diff
@@ -6,6 +6,7 @@ I tried to give credit whenever possible. If I have missed anyone, kindly add it
 
 ### In Development
 
+- Tweak: Add support for Adafruit trinket3/trinket5/protrinket3/protrinket5 by improved BOARDS_TXT parsing
```

Comment:
Sure, I added a commit to fix the HISTORY.md.
Good news you can't break it :-)
Let me know if you prefer a collapsed single commit than the two.